.golangci.yml

```
@@ -70,7 +68,7 @@ linters-settings:
   nolintlint:
     allow-leading-space: true # don't require machine-readable nolint directives (i.e. with no leading space)
     allow-unused: false # report any unused nolint directives
-    require-explanation: false # don't require an explanation for nolint directives
+    equire-explanation: false # don't require an explanation for nolint directives @TODO enable why no lint
```

```suggestion
    require-explanation: false # don't require an explanation for nolint directives @TODO enable why no lint
```

Should not be `require` ?

---

Done